addons/addon-base-rest-api/packages/services/lib/authentication-providers/built-in-providers/cogito-user-pool/user-attributes-mapper-service.js

```diff
@@ -54,14 +58,22 @@ class UserAttributesMapperService extends Service {
       decodedToken.identities &&
       decodedToken.identities[0] &&
       _.toUpper(decodedToken.identities[0].providerType) === 'SAML';
+    if (_.isUndefined(isSamlAuthenticatedUser)) return false;
```

Comment:
I think this can be simplified to be 
`return !_.isUndefined(isSamlAuthenticatedUser) && isSamlAuthenticatedUser`